### ISSUE TITLE
GitHub releases

### ISSUE THREAD

**Issue by mxssl** [Uniswap/unichain-node#13]
Hi! Could you please use GitHub releases? It would be convenient for node runners to monitor new versions of this repo and update their self-hosted Unichain nodes.

**Comment by trianglesphere** [comment]
Yes, will start adding releases. It would also be a good idea to follow upstream optimism releases as those will be more frequent than here.
